lib/roo_on_rails/checks/documentation/playbook_template.md

```diff
@@ -0,0 +1,25 @@
+# Playbook
+
+This document details the failure modes of this service and mitigating steps that can be taken.
+
+_FIXME: go through this document and fill out the sections marked FIXME._
+
+## Responsible humans
+
+As a last resort, in an emergency, the following people have experience working with this service and have claimed responsibility for the running of this service. **If you have executed the plays below** and there is still a clear and present SEV-1 or greater issue, use the links below to get in touch:
+
+- Responsible McPerson ([PagerDuty](https://deliveroo.pagerduty.com/users/FIXME), [Workplace](https://deliveroo.facebook.com/profile.php?id=FIXME))
+
+## Owned Models
+
+- _FIXME_ - a model representing `FIXME`.
+
+## Failure modes
+
+### FIXME: an ability/functional area of the service
```

Comment:
:+1: I'll add something to illustrate that; though there will be some which don't map to an alert (eg. [this one from actor tracking](https://github.com/deliveroo/actor-tracking/blob/master/docs/playbook.md#-disabling-a-feature-flagexperiment-when-florence-is-unavailable))